src/cli/config.js

```diff
@@ -4,6 +4,7 @@ import type {Schema} from '../types';
 import type {GraphQLSchema} from 'graphql/type/schema';
 
 import {schemaFromIntrospectionData} from '../schemaFromIntrospectionData';
+import jsonSchema from './schema.json';
```

Comment:
It's a little bit confusing having a variable named `jsonSchema` while we're using a module named `jsonschema`.  Maybe rename this variable to `schema` or `configSchema`.